molecule/default/molecule.yml

```diff
@@ -0,0 +1,30 @@
+---
+dependency:
+  name: galaxy
+  role-file: requirements.yml
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: centos_image
+    image: centos:7
+    privileged: true
```

Comment:
Is privileged necessary for these two contianers?

Reply:
Have removed